linode/linode_client.py

```diff
@@ -199,6 +199,39 @@ def get_apps(self, *filters):
         return self.client._get_and_filter(AuthorizedApp, *filters)
 
 
+class LongviewGroup(Group):
+    def get_clients(self, *filters):
+        """
+        Requests and returns a paginated list of LongviewClients on your
+        account.
+        """
+        return self.client._get_and_filter(LongviewClient, *filters)
+
+    def create_client(self, label=None):
+        """
+        Creates a new LongviewClient, optinoally with a given label.
+
+        :param label: The label for the new client.  If None, a default label based
+            on the new client's ID will be used.
+
+        :returns: A new LongviewClient
+
+        :raises ApIError: If a non-200 status code is returned
```

Comment:
I dont see this being raised

Reply:
That error is raise in [_api_call](https://github.com/linode/python-linode-api/blob/master/linode/linode_client.py#L442) if the resulting status code indicates an error occurred - by extension, anything the contacts the api can raise it.  Should it not be included here?

Reply:
you good just didnt see that. LGTM